artifact v4 で動くようにする

deploy が次のように失敗するので、以下の URL を参考にして修正する

- https://github.com/actions/upload-artifact

```
2025-02-02T19:48:40.9251860Z Current runner version: '2.322.0'
2025-02-02T19:48:40.9276485Z ##[group]Operating System
2025-02-02T19:48:40.9277341Z Ubuntu
2025-02-02T19:48:40.9277823Z 24.04.1
2025-02-02T19:48:40.9278329Z LTS
2025-02-02T19:48:40.9278864Z ##[endgroup]
2025-02-02T19:48:40.9279355Z ##[group]Runner Image
2025-02-02T19:48:40.9279950Z Image: ubuntu-24.04
2025-02-02T19:48:40.9280524Z Version: 20250126.1.0
2025-02-02T19:48:40.9281514Z Included Software: https://github.com/actions/runner-images/blob/ubuntu24/20250126.1/images/ubuntu/Ubuntu2404-Readme.md
2025-02-02T19:48:40.9283134Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu24%2F20250126.1
2025-02-02T19:48:40.9284101Z ##[endgroup]
2025-02-02T19:48:40.9284610Z ##[group]Runner Image Provisioner
2025-02-02T19:48:40.9285205Z 2.0.422.1
2025-02-02T19:48:40.9285723Z ##[endgroup]
2025-02-02T19:48:40.9286784Z ##[group]GITHUB_TOKEN Permissions
2025-02-02T19:48:40.9288844Z Contents: read
2025-02-02T19:48:40.9289410Z Metadata: read
2025-02-02T19:48:40.9290080Z Pages: write
2025-02-02T19:48:40.9290745Z ##[endgroup]
2025-02-02T19:48:40.9294400Z Secret source: Actions
2025-02-02T19:48:40.9295166Z Prepare workflow directory
2025-02-02T19:48:40.9664954Z Prepare all required actions
2025-02-02T19:48:40.9701293Z Getting action download info
2025-02-02T19:48:41.2014151Z Download action repository 'actions/checkout@v3' (SHA:f43a0e5ff2bd294095638e18286ca9a3d1956744)
2025-02-02T19:48:41.2940382Z Download action repository 'oven-sh/setup-bun@v1' (SHA:f4d14e03ff726c06358e5557344e1da148b56cf7)
2025-02-02T19:48:41.9751451Z Download action repository 'actions/configure-pages@v3' (SHA:b8130d9ab958b325bbde9786d62f2c97a9885a0e)
2025-02-02T19:48:42.7352793Z Download action repository 'actions/upload-artifact@v4' (SHA:65c4c4a1ddee5b72f698fdd19549f0f0fb45cf08)
2025-02-02T19:48:42.8489250Z Download action repository 'actions/deploy-pages@v1' (SHA:f27bcc15848fdcdcc02f01754eb838e44bcf389b)
2025-02-02T19:48:43.6382104Z Complete job name: deploy
2025-02-02T19:48:43.7066246Z ##[group]Run actions/checkout@v3
2025-02-02T19:48:43.7066987Z with:
2025-02-02T19:48:43.7067316Z   repository: yaakaito/yaakaito
2025-02-02T19:48:43.7067887Z   token: ***
2025-02-02T19:48:43.7068207Z   ssh-strict: true
2025-02-02T19:48:43.7068533Z   persist-credentials: true
2025-02-02T19:48:43.7068884Z   clean: true
2025-02-02T19:48:43.7069220Z   sparse-checkout-cone-mode: true
2025-02-02T19:48:43.7069598Z   fetch-depth: 1
2025-02-02T19:48:43.7069911Z   fetch-tags: false
2025-02-02T19:48:43.7070227Z   lfs: false
2025-02-02T19:48:43.7070539Z   submodules: false
2025-02-02T19:48:43.7070860Z   set-safe-directory: true
2025-02-02T19:48:43.7071405Z ##[endgroup]
2025-02-02T19:48:43.8824837Z Syncing repository: yaakaito/yaakaito
2025-02-02T19:48:43.8826298Z ##[group]Getting Git version info
2025-02-02T19:48:43.8826820Z Working directory is '/home/runner/work/yaakaito/yaakaito'
2025-02-02T19:48:43.8827530Z [command]/usr/bin/git version
2025-02-02T19:48:43.8883855Z git version 2.48.1
2025-02-02T19:48:43.8912582Z ##[endgroup]
2025-02-02T19:48:43.8930431Z Temporarily overriding HOME='/home/runner/work/_temp/f53a8170-efb6-45e9-8414-491df1e964b1' before making global git config changes
2025-02-02T19:48:43.8932371Z Adding repository directory to the temporary git global config as a safe directory
2025-02-02T19:48:43.8935285Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/yaakaito/yaakaito
2025-02-02T19:48:43.8967797Z Deleting the contents of '/home/runner/work/yaakaito/yaakaito'
2025-02-02T19:48:43.8971217Z ##[group]Initializing the repository
2025-02-02T19:48:43.8974420Z [command]/usr/bin/git init /home/runner/work/yaakaito/yaakaito
2025-02-02T19:48:43.9067081Z hint: Using 'master' as the name for the initial branch. This default branch name
2025-02-02T19:48:43.9068206Z hint: is subject to change. To configure the initial branch name to use in all
2025-02-02T19:48:43.9068983Z hint: of your new repositories, which will suppress this warning, call:
2025-02-02T19:48:43.9069855Z hint:
2025-02-02T19:48:43.9070501Z hint: 	git config --global init.defaultBranch <name>
2025-02-02T19:48:43.9071513Z hint:
2025-02-02T19:48:43.9072505Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2025-02-02T19:48:43.9073527Z hint: 'development'. The just-created branch can be renamed via this command:
2025-02-02T19:48:43.9074333Z hint:
2025-02-02T19:48:43.9074678Z hint: 	git branch -m <name>
2025-02-02T19:48:43.9075218Z Initialized empty Git repository in /home/runner/work/yaakaito/yaakaito/.git/
2025-02-02T19:48:43.9085719Z [command]/usr/bin/git remote add origin https://github.com/yaakaito/yaakaito
2025-02-02T19:48:43.9116306Z ##[endgroup]
2025-02-02T19:48:43.9117302Z ##[group]Disabling automatic garbage collection
2025-02-02T19:48:43.9120001Z [command]/usr/bin/git config --local gc.auto 0
2025-02-02T19:48:43.9146402Z ##[endgroup]
2025-02-02T19:48:43.9147351Z ##[group]Setting up auth
2025-02-02T19:48:43.9152403Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2025-02-02T19:48:43.9179290Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2025-02-02T19:48:43.9491714Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2025-02-02T19:48:43.9516201Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2025-02-02T19:48:43.9776218Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2025-02-02T19:48:43.9812901Z ##[endgroup]
2025-02-02T19:48:43.9814270Z ##[group]Fetching the repository
2025-02-02T19:48:43.9821530Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +603be7d227130f9b336a9459bf44c5aa93115882:refs/remotes/origin/main
2025-02-02T19:48:44.3919713Z remote: Enumerating objects: 54, done.        
2025-02-02T19:48:44.3920386Z remote: Counting objects:   1% (1/54)        
2025-02-02T19:48:44.3921066Z remote: Counting objects:   3% (2/54)        
2025-02-02T19:48:44.3922042Z remote: Counting objects:   5% (3/54)        
2025-02-02T19:48:44.3922792Z remote: Counting objects:   7% (4/54)        
2025-02-02T19:48:44.3923529Z remote: Counting objects:   9% (5/54)        
2025-02-02T19:48:44.3924297Z remote: Counting objects:  11% (6/54)        
2025-02-02T19:48:44.3925038Z remote: Counting objects:  12% (7/54)        
2025-02-02T19:48:44.3925730Z remote: Counting objects:  14% (8/54)        
2025-02-02T19:48:44.3926406Z remote: Counting objects:  16% (9/54)        
2025-02-02T19:48:44.3927088Z remote: Counting objects:  18% (10/54)        
2025-02-02T19:48:44.3927764Z remote: Counting objects:  20% (11/54)        
2025-02-02T19:48:44.3928342Z remote: Counting objects:  22% (12/54)        
2025-02-02T19:48:44.3928799Z remote: Counting objects:  24% (13/54)        
2025-02-02T19:48:44.3929243Z remote: Counting objects:  25% (14/54)        
2025-02-02T19:48:44.3929664Z remote: Counting objects:  27% (15/54)        
2025-02-02T19:48:44.3930091Z remote: Counting objects:  29% (16/54)        
2025-02-02T19:48:44.3930514Z remote: Counting objects:  31% (17/54)        
2025-02-02T19:48:44.3930930Z remote: Counting objects:  33% (18/54)        
2025-02-02T19:48:44.3931350Z remote: Counting objects:  35% (19/54)        
2025-02-02T19:48:44.3932019Z remote: Counting objects:  37% (20/54)        
2025-02-02T19:48:44.3932444Z remote: Counting objects:  38% (21/54)        
2025-02-02T19:48:44.3932861Z remote: Counting objects:  40% (22/54)        
2025-02-02T19:48:44.3933280Z remote: Counting objects:  42% (23/54)        
2025-02-02T19:48:44.3933707Z remote: Counting objects:  44% (24/54)        
2025-02-02T19:48:44.3934150Z remote: Counting objects:  46% (25/54)        
2025-02-02T19:48:44.3934596Z remote: Counting objects:  48% (26/54)        
2025-02-02T19:48:44.3935266Z remote: Counting objects:  50% (27/54)        
2025-02-02T19:48:44.3935710Z remote: Counting objects:  51% (28/54)        
2025-02-02T19:48:44.3936132Z remote: Counting objects:  53% (29/54)        
2025-02-02T19:48:44.3936548Z remote: Counting objects:  55% (30/54)        
2025-02-02T19:48:44.3936972Z remote: Counting objects:  57% (31/54)        
2025-02-02T19:48:44.3937400Z remote: Counting objects:  59% (32/54)        
2025-02-02T19:48:44.3937896Z remote: Counting objects:  61% (33/54)        
2025-02-02T19:48:44.3938314Z remote: Counting objects:  62% (34/54)        
2025-02-02T19:48:44.3938738Z remote: Counting objects:  64% (35/54)        
2025-02-02T19:48:44.3939160Z remote: Counting objects:  66% (36/54)        
2025-02-02T19:48:44.3939579Z remote: Counting objects:  68% (37/54)        
2025-02-02T19:48:44.3940014Z remote: Counting objects:  70% (38/54)        
2025-02-02T19:48:44.3940446Z remote: Counting objects:  72% (39/54)        
2025-02-02T19:48:44.3940900Z remote: Counting objects:  74% (40/54)        
2025-02-02T19:48:44.3941335Z remote: Counting objects:  75% (41/54)        
2025-02-02T19:48:44.3942031Z remote: Counting objects:  77% (42/54)        
2025-02-02T19:48:44.3942520Z remote: Counting objects:  79% (43/54)        
2025-02-02T19:48:44.3942946Z remote: Counting objects:  81% (44/54)        
2025-02-02T19:48:44.3943359Z remote: Counting objects:  83% (45/54)        
2025-02-02T19:48:44.3943781Z remote: Counting objects:  85% (46/54)        
2025-02-02T19:48:44.3944204Z remote: Counting objects:  87% (47/54)        
2025-02-02T19:48:44.3944625Z remote: Counting objects:  88% (48/54)        
2025-02-02T19:48:44.3945044Z remote: Counting objects:  90% (49/54)        
2025-02-02T19:48:44.3945464Z remote: Counting objects:  92% (50/54)        
2025-02-02T19:48:44.3945878Z remote: Counting objects:  94% (51/54)        
2025-02-02T19:48:44.3946301Z remote: Counting objects:  96% (52/54)        
2025-02-02T19:48:44.3946948Z remote: Counting objects:  98% (53/54)        
2025-02-02T19:48:44.3947416Z remote: Counting objects: 100% (54/54)        
2025-02-02T19:48:44.3947871Z remote: Counting objects: 100% (54/54), done.        
2025-02-02T19:48:44.3948333Z remote: Compressing objects:   2% (1/45)        
2025-02-02T19:48:44.3948776Z remote: Compressing objects:   4% (2/45)        
2025-02-02T19:48:44.3949229Z remote: Compressing objects:   6% (3/45)        
2025-02-02T19:48:44.3949668Z remote: Compressing objects:   8% (4/45)        
2025-02-02T19:48:44.3950099Z remote: Compressing objects:  11% (5/45)        
2025-02-02T19:48:44.3950538Z remote: Compressing objects:  13% (6/45)        
2025-02-02T19:48:44.3950972Z remote: Compressing objects:  15% (7/45)        
2025-02-02T19:48:44.3951407Z remote: Compressing objects:  17% (8/45)        
2025-02-02T19:48:44.3952044Z remote: Compressing objects:  20% (9/45)        
2025-02-02T19:48:44.3952492Z remote: Compressing objects:  22% (10/45)        
2025-02-02T19:48:44.3952939Z remote: Compressing objects:  24% (11/45)        
2025-02-02T19:48:44.3953395Z remote: Compressing objects:  26% (12/45)        
2025-02-02T19:48:44.3953838Z remote: Compressing objects:  28% (13/45)        
2025-02-02T19:48:44.3954277Z remote: Compressing objects:  31% (14/45)        
2025-02-02T19:48:44.3954709Z remote: Compressing objects:  33% (15/45)        
2025-02-02T19:48:44.3955152Z remote: Compressing objects:  35% (16/45)        
2025-02-02T19:48:44.3955580Z remote: Compressing objects:  37% (17/45)        
2025-02-02T19:48:44.3956016Z remote: Compressing objects:  40% (18/45)        
2025-02-02T19:48:44.3956452Z remote: Compressing objects:  42% (19/45)        
2025-02-02T19:48:44.4290726Z remote: Compressing objects:  44% (20/45)        
2025-02-02T19:48:44.4291947Z remote: Compressing objects:  46% (21/45)        
2025-02-02T19:48:44.4292890Z remote: Compressing objects:  48% (22/45)        
2025-02-02T19:48:44.4293787Z remote: Compressing objects:  51% (23/45)        
2025-02-02T19:48:44.4294632Z remote: Compressing objects:  53% (24/45)        
2025-02-02T19:48:44.4295586Z remote: Compressing objects:  55% (25/45)        
2025-02-02T19:48:44.4296080Z remote: Compressing objects:  57% (26/45)        
2025-02-02T19:48:44.4296545Z remote: Compressing objects:  60% (27/45)        
2025-02-02T19:48:44.4297007Z remote: Compressing objects:  62% (28/45)        
2025-02-02T19:48:44.4297472Z remote: Compressing objects:  64% (29/45)        
2025-02-02T19:48:44.4297936Z remote: Compressing objects:  66% (30/45)        
2025-02-02T19:48:44.4298392Z remote: Compressing objects:  68% (31/45)        
2025-02-02T19:48:44.4298850Z remote: Compressing objects:  71% (32/45)        
2025-02-02T19:48:44.4299304Z remote: Compressing objects:  73% (33/45)        
2025-02-02T19:48:44.4299760Z remote: Compressing objects:  75% (34/45)        
2025-02-02T19:48:44.4300216Z remote: Compressing objects:  77% (35/45)        
2025-02-02T19:48:44.4300671Z remote: Compressing objects:  80% (36/45)        
2025-02-02T19:48:44.4301125Z remote: Compressing objects:  82% (37/45)        
2025-02-02T19:48:44.4301741Z remote: Compressing objects:  84% (38/45)        
2025-02-02T19:48:44.4302264Z remote: Compressing objects:  86% (39/45)        
2025-02-02T19:48:44.4302718Z remote: Compressing objects:  88% (40/45)        
2025-02-02T19:48:44.4303174Z remote: Compressing objects:  91% (41/45)        
2025-02-02T19:48:44.4303628Z remote: Compressing objects:  93% (42/45)        
2025-02-02T19:48:44.4304073Z remote: Compressing objects:  95% (43/45)        
2025-02-02T19:48:44.4304539Z remote: Compressing objects:  97% (44/45)        
2025-02-02T19:48:44.4304994Z remote: Compressing objects: 100% (45/45)        
2025-02-02T19:48:44.4305466Z remote: Compressing objects: 100% (45/45), done.        
2025-02-02T19:48:44.4910777Z remote: Total 54 (delta 2), reused 31 (delta 0), pack-reused 0 (from 0)        
2025-02-02T19:48:44.4998555Z From https://github.com/yaakaito/yaakaito
2025-02-02T19:48:44.4999923Z  * [new ref]         603be7d227130f9b336a9459bf44c5aa93115882 -> origin/main
2025-02-02T19:48:44.5025509Z ##[endgroup]
2025-02-02T19:48:44.5026198Z ##[group]Determining the checkout info
2025-02-02T19:48:44.5028551Z ##[endgroup]
2025-02-02T19:48:44.5029218Z ##[group]Checking out the ref
2025-02-02T19:48:44.5033520Z [command]/usr/bin/git checkout --progress --force -B main refs/remotes/origin/main
2025-02-02T19:48:44.5113194Z Switched to a new branch 'main'
2025-02-02T19:48:44.5117081Z branch 'main' set up to track 'origin/main'.
2025-02-02T19:48:44.5123185Z ##[endgroup]
2025-02-02T19:48:44.5156892Z [command]/usr/bin/git log -1 --format='%H'
2025-02-02T19:48:44.5178354Z '603be7d227130f9b336a9459bf44c5aa93115882'
2025-02-02T19:48:44.5362206Z ##[group]Run oven-sh/setup-bun@v1
2025-02-02T19:48:44.5362609Z with:
2025-02-02T19:48:44.5362886Z   bun-version: latest
2025-02-02T19:48:44.5363188Z   no-cache: false
2025-02-02T19:48:44.5363462Z ##[endgroup]
2025-02-02T19:48:44.6461493Z Downloading a new version of Bun: https://bun.sh/download/latest/linux/x64?avx2=true&profile=false
2025-02-02T19:48:45.7748415Z [command]/usr/bin/unzip -o -q /home/runner/work/_temp/3c6c14b3-41e4-4ceb-80b8-6a3a6e43f360
2025-02-02T19:48:46.5013805Z [command]/home/runner/.bun/bin/bun --revision
2025-02-02T19:48:46.5050341Z 1.2.2+c1708ea6a
2025-02-02T19:48:46.5171043Z ##[group]Run bun install
2025-02-02T19:48:46.5171355Z [36;1mbun install[0m
2025-02-02T19:48:46.5204113Z shell: /usr/bin/bash -e {0}
2025-02-02T19:48:46.5204447Z ##[endgroup]
2025-02-02T19:48:46.5299893Z bun install v1.2.2 (c1708ea6)
2025-02-02T19:48:47.2627895Z 
2025-02-02T19:48:47.2630056Z + @astrojs/rss@4.0.10
2025-02-02T19:48:47.2635699Z + astro@5.0.5
2025-02-02T19:48:47.2636069Z + sanitize-html@2.13.1
2025-02-02T19:48:47.2636314Z 
2025-02-02T19:48:47.2636491Z 291 packages installed [734.00ms]
2025-02-02T19:48:47.2805524Z ##[group]Run bun ./scripts/notes-from-issue.ts
2025-02-02T19:48:47.2805935Z [36;1mbun ./scripts/notes-from-issue.ts[0m
2025-02-02T19:48:47.2833084Z shell: /usr/bin/bash -e {0}
2025-02-02T19:48:47.2833318Z env:
2025-02-02T19:48:47.2833668Z   GH_TOKEN: ***
2025-02-02T19:48:47.2834032Z ##[endgroup]
2025-02-02T19:48:47.6897605Z Issues have been saved as Markdown files.
2025-02-02T19:48:47.6966143Z ##[group]Run bun run build
2025-02-02T19:48:47.6966553Z [36;1mbun run build[0m
2025-02-02T19:48:47.7002120Z shell: /usr/bin/bash -e {0}
2025-02-02T19:48:47.7002488Z ##[endgroup]
2025-02-02T19:48:47.7086855Z $ astro build
2025-02-02T19:48:49.1824699Z 19:48:49 [content] Syncing content
2025-02-02T19:48:49.1825434Z 19:48:49 [content] Astro version changed
2025-02-02T19:48:49.1825942Z 19:48:49 [content] Clearing content store
2025-02-02T19:48:49.1844923Z 19:48:49 [content] Synced content
2025-02-02T19:48:49.1850234Z 19:48:49 [types] Generated 112ms
2025-02-02T19:48:49.1853699Z 19:48:49 [build] output: "static"
2025-02-02T19:48:49.1855590Z 19:48:49 [build] directory: /home/runner/work/yaakaito/yaakaito/web/dist/
2025-02-02T19:48:49.1856287Z 19:48:49 [build] Collecting build info...
2025-02-02T19:48:49.1861530Z 19:48:49 [build] ✓ Completed in 132ms.
2025-02-02T19:48:49.1887372Z 19:48:49 [build] Building static entrypoints...
2025-02-02T19:48:50.9970673Z "pagesGlobToRssItems" is imported from external module "@astrojs/rss" but never used in "src/pages/rss.xml.js".
2025-02-02T19:48:51.0892721Z 19:48:51 [vite] [32m✓ built in 1.85s[39m
2025-02-02T19:48:51.0893457Z 19:48:51 [build] ✓ Completed in 1.90s.
2025-02-02T19:48:51.0896610Z 
2025-02-02T19:48:51.0896946Z  building client (vite) 
2025-02-02T19:48:51.0993389Z 19:48:51 [vite] transforming...
2025-02-02T19:48:51.1017905Z 19:48:51 [vite] [32m✓[39m 3 modules transformed.
2025-02-02T19:48:51.1047816Z 19:48:51 [vite] rendering chunks...
2025-02-02T19:48:51.1064229Z 19:48:51 [vite] computing gzip size...
2025-02-02T19:48:51.1078402Z 19:48:51 [vite] [2mdist/[22m[2m_astro/[22m[36mglobal-layout.astro_astro_type_script_index_1_lang.gnFk9-k0.js  [39m[1m[2m0.06 kB[22m[1m[22m[2m │ gzip: 0.07 kB[22m
2025-02-02T19:48:51.1079507Z 19:48:51 [vite] [32m✓ built in 15ms[39m
2025-02-02T19:48:51.1152479Z 
2025-02-02T19:48:51.1152719Z  generating static routes 
2025-02-02T19:48:51.1204933Z 19:48:51 ▶ src/pages/blog/2023/develop-in-codespaces-and-devcontainer.md
2025-02-02T19:48:51.1296455Z 19:48:51   └─ /blog/2023/develop-in-codespaces-and-devcontainer/index.html (+9ms)
2025-02-02T19:48:51.1309016Z 19:48:51 ▶ src/pages/blog/2025/popover-dialog.md
2025-02-02T19:48:51.1330409Z 19:48:51   └─ /blog/2025/popover-dialog/index.html (+2ms)
2025-02-02T19:48:51.1338914Z 19:48:51 ▶ src/pages/blog/index.astro
2025-02-02T19:48:51.1344373Z Astro.glob is deprecated and will be removed in a future major version of Astro.
2025-02-02T19:48:51.1345070Z Use import.meta.glob instead: https://vitejs.dev/guide/features.html#glob-import
2025-02-02T19:48:51.1360725Z 19:48:51   └─ /blog/index.html (+2ms)
2025-02-02T19:48:51.1373376Z 19:48:51 ▶ src/pages/note/1.md
2025-02-02T19:48:51.1396666Z 19:48:51   └─ /note/1/index.html (+2ms)
2025-02-02T19:48:51.1409053Z 19:48:51 ▶ src/pages/note/10.md
2025-02-02T19:48:51.1427303Z 19:48:51   └─ /note/10/index.html (+2ms)
2025-02-02T19:48:51.1438869Z 19:48:51 ▶ src/pages/note/11.md
2025-02-02T19:48:51.1456072Z 19:48:51   └─ /note/11/index.html (+2ms)
2025-02-02T19:48:51.1467659Z 19:48:51 ▶ src/pages/note/12.md
2025-02-02T19:48:51.1483408Z 19:48:51   └─ /note/12/index.html (+2ms)
2025-02-02T19:48:51.1494231Z 19:48:51 ▶ src/pages/note/13.md
2025-02-02T19:48:51.1531347Z 19:48:51   └─ /note/13/index.html (+4ms)
2025-02-02T19:48:51.1542863Z 19:48:51 ▶ src/pages/note/14.md
2025-02-02T19:48:51.1559195Z 19:48:51   └─ /note/14/index.html (+2ms)
2025-02-02T19:48:51.1570504Z 19:48:51 ▶ src/pages/note/15.md
2025-02-02T19:48:51.1587740Z 19:48:51   └─ /note/15/index.html (+2ms)
2025-02-02T19:48:51.1601054Z 19:48:51 ▶ src/pages/note/2.md
2025-02-02T19:48:51.1617994Z 19:48:51   └─ /note/2/index.html (+2ms)
2025-02-02T19:48:51.1630497Z 19:48:51 ▶ src/pages/note/25.md
2025-02-02T19:48:51.1646013Z 19:48:51   └─ /note/25/index.html (+2ms)
2025-02-02T19:48:51.1657957Z 19:48:51 ▶ src/pages/note/26.md
2025-02-02T19:48:51.1673485Z 19:48:51   └─ /note/26/index.html (+1ms)
2025-02-02T19:48:51.1684838Z 19:48:51 ▶ src/pages/note/28.md
2025-02-02T19:48:51.1699429Z 19:48:51   └─ /note/28/index.html (+1ms)
2025-02-02T19:48:51.1711511Z 19:48:51 ▶ src/pages/note/3.md
2025-02-02T19:48:51.1725139Z 19:48:51   └─ /note/3/index.html (+1ms)
2025-02-02T19:48:51.1736367Z 19:48:51 ▶ src/pages/note/4.md
2025-02-02T19:48:51.1748146Z 19:48:51   └─ /note/4/index.html (+1ms)
2025-02-02T19:48:51.1758902Z 19:48:51 ▶ src/pages/note/5.md
2025-02-02T19:48:51.1770666Z 19:48:51   └─ /note/5/index.html (+1ms)
2025-02-02T19:48:51.1781350Z 19:48:51 ▶ src/pages/note/6.md
2025-02-02T19:48:51.1793609Z 19:48:51   └─ /note/6/index.html (+1ms)
2025-02-02T19:48:51.1805355Z 19:48:51 ▶ src/pages/note/8.md
2025-02-02T19:48:51.1818076Z 19:48:51   └─ /note/8/index.html (+1ms)
2025-02-02T19:48:51.1829961Z 19:48:51 ▶ src/pages/note/9.md
2025-02-02T19:48:51.1844570Z 19:48:51   └─ /note/9/index.html (+1ms)
2025-02-02T19:48:51.1852447Z 19:48:51 ▶ src/pages/note/index.astro
2025-02-02T19:48:51.1858611Z Astro.glob is deprecated and will be removed in a future major version of Astro.
2025-02-02T19:48:51.1859644Z Use import.meta.glob instead: https://vitejs.dev/guide/features.html#glob-import
2025-02-02T19:48:51.1883676Z 19:48:51   └─ /note/index.html (+3ms)
2025-02-02T19:48:51.2260692Z 19:48:51 λ src/pages/rss.xml.js
2025-02-02T19:48:51.2643068Z 19:48:51   └─ /rss.xml (+38ms)
2025-02-02T19:48:51.2654754Z 19:48:51 ▶ src/pages/index.astro
2025-02-02T19:48:51.2659793Z Astro.glob is deprecated and will be removed in a future major version of Astro.
2025-02-02T19:48:51.2660561Z Use import.meta.glob instead: https://vitejs.dev/guide/features.html#glob-import
2025-02-02T19:48:51.2663048Z Astro.glob is deprecated and will be removed in a future major version of Astro.
2025-02-02T19:48:51.2663817Z Use import.meta.glob instead: https://vitejs.dev/guide/features.html#glob-import
2025-02-02T19:48:51.2686356Z 19:48:51   └─ /index.html (+3ms)
2025-02-02T19:48:51.2686920Z 19:48:51 ✓ Completed in 154ms.
2025-02-02T19:48:51.2687230Z 
2025-02-02T19:48:51.2733998Z 19:48:51 [build] 22 page(s) built in 2.22s
2025-02-02T19:48:51.2734400Z 19:48:51 [build] Complete!
2025-02-02T19:48:51.2978980Z ##[group]Run actions/configure-pages@v3
2025-02-02T19:48:51.2979259Z with:
2025-02-02T19:48:51.2979543Z   token: ***
2025-02-02T19:48:51.2979718Z   enablement: false
2025-02-02T19:48:51.2979895Z ##[endgroup]
2025-02-02T19:48:51.7465180Z ##[group]Run actions/upload-artifact@v4
2025-02-02T19:48:51.7465464Z with:
2025-02-02T19:48:51.7465632Z   name: pages-artifact
2025-02-02T19:48:51.7465830Z   path: ./web/dist
2025-02-02T19:48:51.7466020Z   if-no-files-found: warn
2025-02-02T19:48:51.7466223Z   compression-level: 6
2025-02-02T19:48:51.7466409Z   overwrite: false
2025-02-02T19:48:51.7466595Z   include-hidden-files: false
2025-02-02T19:48:51.7466795Z env:
2025-02-02T19:48:51.7466953Z   GITHUB_PAGES: true
2025-02-02T19:48:51.7467135Z ##[endgroup]
2025-02-02T19:48:51.9833074Z With the provided path, there will be 27 files uploaded
2025-02-02T19:48:51.9840072Z Artifact name is valid!
2025-02-02T19:48:51.9841385Z Root directory input is valid!
2025-02-02T19:48:52.2385418Z Beginning upload of artifact content to blob storage
2025-02-02T19:48:52.6446827Z Uploaded bytes 86901
2025-02-02T19:48:52.7112890Z Finished uploading artifact content to blob storage!
2025-02-02T19:48:52.7116262Z SHA256 hash of uploaded artifact zip is 28e5505038ac716ac4174ada86dddf8b38ae3ce33155caf6c5f5e41168602cd6
2025-02-02T19:48:52.7117532Z Finalizing artifact upload
2025-02-02T19:48:52.8292862Z Artifact pages-artifact.zip successfully finalized. Artifact ID 2523790456
2025-02-02T19:48:52.8293918Z Artifact pages-artifact has been successfully uploaded! Final size is 86901 bytes. Artifact ID is 2523790456
2025-02-02T19:48:52.8300900Z Artifact download URL: https://github.com/yaakaito/yaakaito/actions/runs/13101924173/artifacts/2523790456
2025-02-02T19:48:52.8413761Z ##[group]Run actions/deploy-pages@v1
2025-02-02T19:48:52.8414033Z with:
2025-02-02T19:48:52.8414194Z   emit_telemetry: false
2025-02-02T19:48:52.8414679Z   token: ***
2025-02-02T19:48:52.8414849Z   timeout: 600000
2025-02-02T19:48:52.8415018Z   error_count: 10
2025-02-02T19:48:52.8415200Z   reporting_interval: 5000
2025-02-02T19:48:52.8415414Z   artifact_name: github-pages
2025-02-02T19:48:52.8415625Z   preview: false
2025-02-02T19:48:52.8415794Z env:
2025-02-02T19:48:52.8415948Z   GITHUB_PAGES: true
2025-02-02T19:48:52.8416132Z ##[endgroup]
2025-02-02T19:48:53.2075904Z Artifact exchange URL: https://pipelinesghubeus7.actions.githubusercontent.com/5PBU7b0MNQOM4nbc1nrqx3kh7sqKbQ6jJ0O62HA3kvYZjJzHhB/_apis/pipelines/workflows/13101924173/artifacts?api-version=6.0-preview
2025-02-02T19:48:53.3044560Z ##[error]Error: No uploaded artifact was found! Please check if there are any errors at build step, or uploaded artifact name is correct.
    at getSignedArtifactUrl (/home/runner/work/_actions/actions/deploy-pages/v1/webpack:/deploy-pages/src/api-client.js:93:1)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Deployment.create (/home/runner/work/_actions/actions/deploy-pages/v1/webpack:/deploy-pages/src/deployment.js:53:1)
    at main (/home/runner/work/_actions/actions/deploy-pages/v1/webpack:/deploy-pages/src/index.js:30:1)
2025-02-02T19:48:53.3053328Z ##[error]Error: No uploaded artifact was found! Please check if there are any errors at build step, or uploaded artifact name is correct.
2025-02-02T19:48:53.3134134Z Post job cleanup.
2025-02-02T19:48:53.3845049Z [command]/usr/bin/git version
2025-02-02T19:48:53.3879350Z git version 2.48.1
2025-02-02T19:48:53.3920651Z Temporarily overriding HOME='/home/runner/work/_temp/6bab1d72-6e4e-4c2a-baf8-c32ddb9e9f98' before making global git config changes
2025-02-02T19:48:53.3921789Z Adding repository directory to the temporary git global config as a safe directory
2025-02-02T19:48:53.3924588Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/yaakaito/yaakaito
2025-02-02T19:48:53.3955354Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2025-02-02T19:48:53.3982904Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2025-02-02T19:48:53.4208102Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2025-02-02T19:48:53.4227560Z http.https://github.com/.extraheader
2025-02-02T19:48:53.4239089Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2025-02-02T19:48:53.4271504Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2025-02-02T19:48:53.4586287Z Evaluate and set environment url
2025-02-02T19:48:53.4590180Z Evaluated environment url: 
2025-02-02T19:48:53.4590841Z Cleaning up orphan processes
```

--- a/.github/workflows/static.yml
+++ b/.github/workflows/static.yml
@@ -51,7 +51,9 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: pages-artifact
-          path: './web/dist'
+          path: 'web/dist'
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v1
+        with:
+          artifact_name: pages-artifact

